src/sentry/taskworker/worker.py

```diff
@@ -158,7 +167,18 @@ def process_task(self, activation: TaskActivation) -> TaskActivation | None:
                 status=TASK_ACTIVATION_STATUS_FAILURE,
             )
 
-        # TODO(taskworker): Add at_most_once checks
+        if activation.retry_state.at_most_once:
```

Comment:
How will tasks indicate that they should be executed at_most_once? In the docs it was outlined as a parameter to the `@register()` call, so we could read the `Task` instance to find out if a task has at_most_once mode enabled as well.

Reply:
OK I'll look at that, that sounds like a cleaner solution.